### DISCUSSION TITLE
Auth0 -  OPError: access_denied from callback

### DISCUSSION THREAD

**Discussion by hobadams** [nextauthjs/next-auth#7631] [Help]
Hey,

I'm hoping this is a simple misconfiguration.

I'm using the Auth0 provider for authorisation to my app.  I get correctly redirected to Auth0 to login and can see the logins are successful in the Auth0 logs.

When i'm redirected to my app I get an `OAUTH_CALLBACK_ERROR` (`OPError: access_denied (Unauthorized)`).

The docs suggest it could be tied to the `state` or `code_verifier` but I can't see how to check or fix this. I certainly feels like the browser cookies aren't playing nice with the server session.

After some debugging I can see this has some connection with the OpenId client package.

Any help would be **hugely** appreciated!

My config are as follows:

**package.json**
```
"next": "13.4.1",
"next-auth": "^4.22.1"
```

**.env.local**
```
NEXTAUTH_URL=http://localhost:3002
SECRET=[MY_SECRET]
AUTH0_CLIENT_ID=[CLIENT_ID]
AUTH0_CLIENT_SECRET=[SECRET]
AUTH0_ISSUER=[ISSUER_URL_WITH_TRAILING_SLASH]
```

**auth.ts**
```
{
session: {
    strategy: 'jwt',
  },
  secret: process.env.SECRET,
  debug: process.env.NODE_ENV === 'development',
  providers: [
    Auth0Provider({
      clientId: process.env.AUTH0_CLIENT_ID || '',
      clientSecret: process.env.AUTH0_CLIENT_SECRET || '',
      issuer: process.env.AUTH0_ISSUER,
      wellKnown: `${process.env.AUTH0_ISSUER}.well-known/openid-configuration`,
    }),
  ],
}
```

**Comment by hobadams** [comment]
I stupidly had wrong config. sorry

**Reply by admin-cohesionx** [reply]
Hi, I've got the same issue, please shed light on what the issue was?
